Makefile.authx

```diff
@@ -51,6 +74,43 @@ init-authx: mkdir
 	source ${PWD}/lib/keycloak/keycloak_setup.sh; \
 	export SERVICE=tyk; \
 	${PWD}/lib/tyk/tyk_setup.sh; \
-	echo  ; \
-	$(MAKE) compose-tyk \
-	${PWD}/lib/tyk/tyk_key_generation.sh;
+	echo ; \
+	$(MAKE) compose-tyk; \
```

Comment:
Deployment needs to support single node (compose) and multi-node (swarm/stack)

Reply:
Can I choose to do the swarm later?  How critical is that for this _release_?